src/plugins/plugin.text_selection.js

```diff
@@ -306,3 +448,190 @@ export class BookreaderWithTextSelection extends BookReader {
 }
 window.BookReader = BookreaderWithTextSelection;
 export default BookreaderWithTextSelection;
+
+
+/**
+ * @param {HTMLElement} parentEl
+ * @param {string} selector
+ * @returns {Map<Element, Rect>}
+ */
+function determineRealRects(parentEl, selector) {
+  const initals = {
+    position: parentEl.style.position,
+    visibility: parentEl.style.visibility,
+    top: parentEl.style.top,
+    left: parentEl.style.left,
+    transform: parentEl.style.transform,
+  };
+  parentEl.style.position = 'absolute';
```

Comment:
q: can these default style be housed in a css class?

Reply:
These are temporary ; they're only used while the element is added to the page to determine sizings, and then quickly removed. I'm afraid adding it to CSS might create some timing headaches!